### PR TITLE
thumbsreducer tests, uuids, indeces for questions, clean up package.json

### PR DESCRIPTION
**Pull request by AriLFrankel**

### CODE REVIEW

**Comment by JTMcDuff** [comment]
Looks good to me.  Need code review.

File: test/thumbsReducerTest.js [hunk 1]
```diff
@@ -31,11 +31,21 @@ describe('thumbsReducer', function() {
     });
   });
 
-  xdescribe('THUMB_CLICKED', function () {
-    it('it should increment the specific thumbType included in the action', function() {
+  describe('THUMB_CLICKED', function () {
+    it('it should increment the specific thumbType included in the action - up', function() {
```
Comment (rtopol2) [inline test/thumbsReducerTest.js]: should change this to say thumbChoice

File: test/thumbsReducerTest.js [hunk 2]
```diff
@@ -31,11 +31,21 @@ describe('thumbsReducer', function() {
...
       assert.deepEqual({topicId: '', displayed: false, up: 1, down: 0, side: 0},
         thumbsReducer(testState,
-        {type: 'THUMB_CLICKED', thumbType: 'up'}));
+        {type: 'THUMB_CLICKED', thumbChoice: 'up'}));
+    });
+    it('it should increment the specific thumbType included in the action - down', function() {
```
Comment (rtopol2) [inline test/thumbsReducerTest.js]: should say thumbChoice instead of thumbType

File: test/thumbsReducerTest.js [hunk 3]
```diff
@@ -31,11 +31,21 @@ describe('thumbsReducer', function() {
...
+      assert.deepEqual({topicId: '', displayed: false, up: 0, down: 1, side: 0},
+        thumbsReducer(testState,
+        {type: 'THUMB_CLICKED', thumbChoice: 'down'}));
+    });
+    it('it should increment the specific thumbType included in the action - side', function() {
```
Comment (rtopol2) [inline test/thumbsReducerTest.js]: should say thumbChoice instead of thumbType

**Comment by JTMcDuff** [comment]
Merging after review.

**Pull request merged**